charts/topolvm/Chart.yaml

```diff
@@ -12,7 +12,7 @@ sources:
 dependencies:
   - name: cert-manager
     repository: https://charts.jetstack.io
-    version: ~1.12.x
+    version: ~1.17.x
```

Comment:
Although the reason for the update is included in the commit, I think it would be better to document the LTS policy.

Reply:
I updated a document.
https://github.com/topolvm/topolvm/compare/5e420cf6b058a35a73b212d288bfd4500b4275cb..a118e73b9b96429496106e2cd54ad3c5cb9ec088